lib/elixir/lib/enum.ex

```diff
@@ -995,9 +995,15 @@ defmodule Enum do
   @doc """
   Similar to `find/3`, but returns the value of the function
   invocation instead of the element itself.
+  An element is considered to be found when the result ist truthy (!= nil or != false).
```

Comment:
There needs to be a newline between the first sentence and the rest of the docs since the first sentence is shown as a summary in the docs :)

```suggestion

  An element is considered to be found when the result is truthy (neither `nil` nor `false`).
```

Reply:
👍